packages/host/app/components/editor/directory.gts

```diff
@@ -113,6 +117,18 @@ export default class Directory extends Component<Args> {
         transform: rotate(-90deg);
       }
 
+      .directory[disabled] {
+        color: var(--boxel-dark);
+      }
+
+      .directory[disabled]:hover {
+        background: transparent;
+      }
+
+      .directory[disabled] :deep(.icon) {
+        --icon-color: var(--boxel-light-600);
+      }
```

Comment:
Are these still needed?

Reply:
they are indeed abandoned, removed now, thanks … though the push is not showing here 🧐